django/utils/autoreload.py

```diff
@@ -232,11 +232,15 @@ def get_child_arguments():
         exe_entrypoint = py_script.with_suffix('.exe')
         if exe_entrypoint.exists():
             # Should be executed directly, ignoring sys.executable.
-            return [exe_entrypoint, *sys.argv[1:]]
+            # TODO: Remove str() when dropping support for PY37.
+            # arg parameter accepts path-like on Windows from Python 3.8.
+            return [str(exe_entrypoint), *sys.argv[1:]]
         script_entrypoint = py_script.with_name('%s-script.py' % py_script.name)
         if script_entrypoint.exists():
             # Should be executed as usual.
-            return [*args, script_entrypoint, *sys.argv[1:]]
+            # TODO: Remove str() when dropping support for PY37.
+            # arg parameter accepts path-like on Windows from Python 3.8.
```

Comment:
```suggestion
            # args parameter accepts path-like on Windows from Python 3.8.
```
The first `subprocess.run()`'s parameter is called [`args`](https://docs.python.org/3/library/subprocess.html#subprocess.run).